contracts/README.md

```diff
@@ -0,0 +1,125 @@
+# Aragon OSx Commons Contracts
+
+This package contains the Solidity smart contracts to be used by the OSx framework and for plugin development.
+For audit information, refer to the ['Audits' section in the root level README.md](../README.md#audits).
+
+## Project
+
+The `contracts` folder includes the following Solidity contracts and libraries:
+
+```markdown
+.
+├── dao
+│ └── IDAO.sol
+├── permission
+│ ├── PermissionLib.sol
+│ ├── auth
+│ │ ├── DaoAuthorizable.sol
+│ │ ├── DaoAuthorizableUpgradeable.sol
+│ │ └── auth.sol
+│ └── condition
+│ ├── IPermissionCondition.sol
+│ ├── PermissionCondition.sol
+│ └── PermissionConditionUpgradeable.sol
+├── plugin
+│ ├── IPlugin.sol
+│ ├── Plugin.sol
+│ ├── PluginCloneable.sol
+│ ├── PluginUUPSUpgradeable.sol
+│ ├── extensions
+│ │ ├── governance
+│ │ │ └── Addresslist.sol
+│ │ ├── membership
+│ │ │ └── IMembership.sol
+│ │ └── proposal
+│ │ ├── IProposal.sol
+│ │ ├── Proposal.sol
+│ │ └── ProposalUpgradeable.sol
+│ └── setup
+│ ├── IPluginSetup.sol
+│ ├── PluginSetup.sol
+│ └── PluginUpgradeableSetup.sol
+└── utils
+├── deployment
+│ ├── ProxyFactory.sol
+│ └── ProxyLib.sol
+├── math
+│ ├── BitMap.sol
+│ ├── Ratio.sol
+│ └── UncheckedMath.sol
+└── versioning
+├── IProtocolVersion.sol
+├── ProtocolVersion.sol
+└── VersionComparisonLib.sol
```

Comment:
this tree could go out of date easily
it would have been nice to generate this markdown automatically and then reference it here

Reply:
True. Or, alternatively, just show the folders.